src/test/node/test.js

```diff
@@ -1,17 +1,14 @@
-'use strict';
-
 import test from 'tape';
 
-test('A passing test', (assert) => {
-  assert.pass('This test will pass.');
-  assert.end();
+test('A passing test', (t) => {
```

Comment:
Could you leave comments above each of these tests that these are just sample tests?